gvm/protocols/gmpv9/gmpv9.py

```diff
@@ -53,23 +53,7 @@ def _check_event(
     event: AlertEvent, condition: AlertCondition, method: AlertMethod
 ):
     if event == AlertEvent.TASK_RUN_STATUS_CHANGED:
-        if not condition:
-            raise RequiredArgument(
-                "condition is required for event {}".format(event.name)
-            )
-
-        if not method:
-            raise RequiredArgument(
-                "method is required for event {}".format(event.name)
-            )
-
-        if condition not in (
-            AlertCondition.ALWAYS,
-            AlertCondition.FILTER_COUNT_CHANGED,
-            AlertCondition.FILTER_COUNT_AT_LEAST,
-            AlertCondition.SEVERITY_AT_LEAST,
-            AlertCondition.SEVERITY_CHANGED,
-        ):
```

Comment:
I really don't know why this method has been changed. Should have been put into another PR.